internal/pkg/install/install.go

```diff
@@ -56,7 +64,7 @@ func Install(args string, data *userdata.UserData) (err error) {
 			switch {
 			case strings.HasPrefix(artifact, "http"):
 				var u *url.URL
-				log.Printf("Downloading %s\n", artifact)
+				log.Printf("downloading %s\n", artifact)
```

Comment:
Most of the rest of the codebase uses lowercase. Figured I'd align this with that convention and we can discuss a style we agree on and refactor later.